lib/sensors/temperature.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "sensors.hpp"
+
+#include <cstdint>
+#include <cstdio>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/i2c.hpp>
+
+namespace hyped::sensors {
+
+const std::uint8_t CTRL          = 0x04;
```

Comment:
rename all the constants to start with k

Reply:
and `static constexpr`